.devops/templates/tools.yml

```diff
@@ -5,9 +5,6 @@ steps:
       versionSpec: '12.x'
     displayName: 'Install Node.js'
 
-  - task: YarnInstaller@0
-    displayName: 'Use Yarn 1.22.x'
-    inputs:
-      versionSpec: 1.22.x
-      checkLatest: false
-      includePrerelease: false
+  - script: |
+      npm i -g yarn@1
```

Comment:
Do we need this if we store the binary in the repo?

Reply:
we do, otherwise `yarn` would not be within PATH, so invoking `yarn` would throw.

Reply:
![image](https://user-images.githubusercontent.com/1223799/109485295-ccc89780-7a81-11eb-9b78-b2785724b5ea.png)

Reply:
> No need to install on CI

I don't get this statement then since we need to install yarn on CI anyway

Reply:
forgot to delete that, thx